### DISCUSSION TITLE
Clarity reagrding CPT

### DISCUSSION THREAD

**Discussion by LovleenKaur-tal** [snigi-gupta/AttendingUniversityAtBuffalo#31] [Q&A] [answered]
I have a question regarding if the university gives an option to pursue classes for Fall online and Spring in person, then will we be eligible for CPT?

**Comment by h8pathak** [comment, marked as answer]
Last year (ie. Fall 2020), students were given an option of Practicum Track which allowed them to begin CPT before one academic year of full-time enrollment.
